src/messages/MOSDECSubOpReadReply.h

```diff
@@ -29,7 +29,7 @@ class MOSDECSubOpReadReply : public MOSDFastDispatchOp {
   ECSubReadReply op;
 
   int get_cost() const override {
-    return 0;
+    return data.length();
```

Comment:
Reviewer feedback requested:

MOSDECSubOpReadReply and MOSDECSubOpWrite now encode write data into the data bl of the message, and now return the length of this buffer for get_cost(), previously these messages returned a cost of 0. This will change how these messages are prioritized by the scheduler. Should they return a cost of 0 instead to preserve old behavior, or was the old behavior a bug?

An aside is that all the ECSubOp* messages and the RepOp message unnecessarily override get_cost when the default method which returns data.length() would provide the same behavior - should these unnecessary overrides be deleted?